src/components/pages/ProductionSettings.vue

```diff
@@ -21,6 +26,10 @@
       </ul>
     </div>
 
+    <div class="tab" v-show="isActiveTab('brief')">
+      <ProductionBrief :value="currentProduction" />
```

Comment:
The value is not needed because the current production is available through mapGetters function.